roofit/roofitcore/src/RooAbsPdf.cxx

```diff
@@ -2289,7 +2289,17 @@ RooDataSet *RooAbsPdf::generate(const RooArgSet &whatVars, const RooDataSet& pro
   }
 }
 
+////////////////////////////////////////////////////////////////////////////////
+
+std::string RooAbsPdf::translate(RooFit::Detail::CodeSquashContext &ctxManager) const
```

Comment:
```suggestion
std::string RooAbsPdf::translate(RooFit::Detail::CodeSquashContext &ctx) const
```
Not that "Manager" got dropped from the class name, it does not make sense to have it in the parameter name either. I would suggest `"ctx"` which is also nice and short.

Probably you can just automatically replace `ctxManager` with `ctx` in the roofit subdirectory of the ROOT repo.